csp-mixin/panels/gcpvpc.libsonnet

```diff
@@ -0,0 +1,134 @@
+local g = import '../g.libsonnet';
+local commonlib = import 'common-lib/common/main.libsonnet';
+{
+  new(this): {
+    _serviceTableCommon(valueColNiceName)::
+      g.panel.table.queryOptions.withTransformations([
+        g.panel.table.queryOptions.transformation.withId('filterFieldsByName')
+        + g.panel.table.queryOptions.transformation.withOptions({
+          include: {
+            pattern: 'service_name.*|Value.*',
+          },
+        }),
+        g.panel.table.queryOptions.transformation.withId('joinByField')
+        + g.panel.table.queryOptions.transformation.withOptions({
+          byField: 'service_name',
+          mode: 'outer',
+        }),
+      ])
+      + g.panel.table.standardOptions.withOverrides([
+        {
+          matcher: {
+            id: 'byName',
+            options: 'service_name',
+          },
+          properties: [
+            {
+              id: 'displayName',
+              value: 'Service name',
+            },
+          ],
+        },
+        {
+          matcher: {
+            id: 'byName',
+            options: 'Value',
+          },
+          properties: [
+            {
+              id: 'displayName',
+              value: valueColNiceName,
+            },
+          ],
+        },
+      ]),
+
+
+    gcpvpc_services_in_use_count:
+      this.signals.gcpvpc.gcpvpc_services_in_use_count.asStat()
+      + g.panel.stat.standardOptions.color.withMode('fixed')
+      + g.panel.stat.standardOptions.color.withFixedColor('text'),
+
+    gcpvpc_subnets_in_use_count:
+      this.signals.gcpvpc.gcpvpc_subnets_in_use_count.asStat()
+      + g.panel.stat.standardOptions.color.withMode('fixed')
+      + g.panel.stat.standardOptions.color.withFixedColor('text'),
+
+    gcpvpc_fixed_usage_tier_network_egress:
+      this.signals.gcpvpc.gcpvpc_fixed_usage_tier_network_egress.asTimeSeries()
+      + g.panel.timeSeries.standardOptions.withNoValue('0'),
+
+    gcpvpc_service_topk_throughput:
+      this.signals.gcpvpc.gcpvpc_service_topk_throughput_bytes.common
+      + g.panel.table.standardOptions.withUnit('bps')
```

Comment:
isn't this unit mentioned in the signal as well?

Reply:
I think, even if you use **asTable** the units are not propagated for any rare reason. This is happening just on tables. Maybe is an error on the common lib.

Reply:
Removing it from the signal would make sense!